aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -110,7 +113,9 @@ impl VMChangeSet {
                 // TODO(aggregator) While everything else must be a resource, first
                 // version of aggregators is implemented as a table item. Revisit when
                 // we split MVHashMap into data and aggregators.
-                resource_write_set.insert(state_key, write_op);
+                // TODO: Currently using MoveTypeLayout as None indicating no aggregators are in
```

Comment:
I might have missed it, looking through the PR, but where do we check if MoveTypeLayout has aggregator/snapshot in it , and passing None if it doesn't?

Reply:
Inside the VM, when we build the change set